sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -1,17 +1,15 @@
 # Release History
 
-## 4.5.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 4.5.0-beta.3 (2025-04-08)
 
 ### Bugs Fixed
 
 - Allow the `ApiVersion` field within `KeyClientOptions` to be settable.
 
 ### Other Changes
 
+- Use generated code to replace internal functionality.

```

Comment:
```suggestion
- Use generated code to replace hand written client.
```

Reply:
I like George's wording more, because at this time we are not replacing the client with generated API, we are only replacing implementation, which is internals.